src/objects/Course_Offering__c.object

```diff
@@ -167,4 +176,5 @@
     <pluralLabel>Course Offerings</pluralLabel>
     <searchLayouts/>
     <sharingModel>ControlledByParent</sharingModel>
+    <visibility>Public</visibility>
```

Comment:
Available since 34.0, this is fine and expected. If it wasn't Public, the object wouldn't be exposed to our customers.